README.md

```diff
@@ -1 +1,18 @@
-# Welcome to OpenBC Website
\ No newline at end of file
+# Welcome to OpenBC
+
+## Project Set Up Instructions
+
+1. Clone the repo `openbc-web` using SSH
+2. `cd` into the repo
+3. Type `node -v` to ensure you have the latest version of Node.js installed, install Node.js or update your version if needed
+4. From the top-level directory of the project directory `run npm install` to install the dev dependencies
+5. Type `npm run start` to start the application (development mode)
+6. To access the application navigate to `http://localhost:3000` in your web browser
+7. To exit the server `Command + C`
```

Comment:
this is different on different machines. on mac it's command + c on linux it's control c. on windows it might be something else. i am not sure if it's important enough to mention this. 

also reference the available commands in package.json.  i.e. for full list of commands, take a look at package.json.